test/integration/dialects/postgres/connection-manager.test.js

```diff
@@ -25,12 +25,5 @@ if (dialect.match(/^postgres/)) {
     it('should correctly parse the moment based timezone', function() {
       return checkTimezoneParsing(this.sequelize.options);
     });
-
-    it('should correctly parse the moment based timezone while fetching hstore oids', function() {
-      // reset oids so we need to refetch them
```

Comment:
No need to reset, this will be reset when creating a new sequelize instance, with a new connection manager